packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -273,6 +296,7 @@ class AnalyzeCommand extends FlutterCommand {
       new RegExp(r'\[lint\] Prefer using lowerCamelCase for constant names.'), // sometimes we have no choice (e.g. when matching other platforms)
       new RegExp(r'\[lint\] Avoid defining a one-member abstract class when a simple function will do.'), // too many false-positives; code review should catch real instances
       new RegExp(r'[0-9]+ (error|warning|hint|lint).+found\.'),
+      new RegExp(r'\[info\] TODO.+'),
```

Comment:
No, these have been around for a long time. These are from TODO: comments in the source code - you're not seeing them?